config/locales/embargoes.en.yml

```diff
@@ -0,0 +1,26 @@
+en:
+  hyrax:
+    embargoes:
```

Comment:
Discussed with @mjgiarlo in slack - decided to incorporate embargo etc. into `hyrax.en.yml` in this PR and open a new issue to restructure the local files (later).